src/ScanPatron/ScanForm.test.js

```diff
@@ -115,4 +119,31 @@ describe('ScanForm', () => {
       });
     });
   });
+
+  describe('when no reading rooms are available at the current service point', () => {
+    beforeEach(() => {
+      useReadingRoom
+        .mockClear()
+        .mockReturnValue({
+          data: {
+            readingRooms: []
+          },
+          refetch: jest.fn(),
+          isLoading: false,
+        });
+      renderComponent(props);
+    });
+
+    it('should display text to indicate no reading rooms defined at the current service point', () => {
+      expect(screen.getByText('No reading rooms defined for the current service point')).toBeDefined();
+    });
+
+    it('should disable input field', () => {
+      expect(screen.getByRole('textbox')).toHaveAttribute('disabled');
```

Comment:
I think you can use `toBeDisabled` here instead

Reply:
As discussed, I am retaining this.